build.proj

```diff
@@ -12,6 +12,17 @@
     <RestoreDuringBuild Condition="'$(RestoreDuringBuild)'==''">true</RestoreDuringBuild>
   </PropertyGroup>
 
+  <!--
+    Load the UpdatePublishedVersions VersionTools task that was just built, if there is one. This
+    allows BuildTools builds to use the current task even when running on a very old BuildTools
+    toolset. Specifically, the TLS 1.2 fix is required.
+
+    TODO: Remove this workaround once BuildTools uses a new BuildTools version: https://github.com/dotnet/buildtools/issues/1934
+  -->
+  <UsingTask TaskName="UpdatePublishedVersions"
```

Comment:
this could be weird if there's every a breaking change in the task's inputs/outputs but I'm guessing 1934 will be fixed sooner than that.

Reply:
Yeah, if that happens we can just override the target, too. I don't expect any breaks though, only new targets for new scenarios.